src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1746,5 +1746,78 @@ internal override System.Reflection.MethodImplAttributes ImplementationAttribute
 
         internal override int TryGetOverloadResolutionPriority()
             => GetEarlyDecodedWellKnownAttributeData()?.OverloadResolutionPriority ?? 0;
+
```

Comment:
These methods were copy/pasted from `SourceMethodSymbol.cs` with no changes to their content.